bin/varnishd/cache/cache_expire.c

```diff
@@ -219,6 +219,32 @@ EXP_Insert(struct worker *wrk, struct objcore *oc)
 	}
 }
 
+/*--------------------------------------------------------------------
+ * Reduce object timers
+ */
+
+void
+EXP_Reduce(struct objcore *oc, vtim_real now,
+    vtim_real ttl, vtim_real grace, vtim_real keep)
+{
+	float next_ttl;
```

Comment:
`vtim_dur` for everything except `now`?

Reply:
Ah, I will change `ttl`, `grace` and `keep` to `vtim_dur`. As for `next_ttl`, looks like this local variable can be eliminated now